forward-message/.env

```diff
@@ -1,8 +1,8 @@
-# description: Messages sent to your Twilio number will get forwarded to this E.164-formatted phone number
-# format: phone_number
+# description: A list of numbers in E.164 format you want to forward incoming messages to, separated by commas
+# format: list(phone_number)
 # required: true
 # link: https://www.twilio.com/docs/glossary/what-e164
-MY_PHONE_NUMBER=+12223334444
+FORWARDING_NUMBERS=+12223334444,+491761234567
```

Comment:
German number is still here.

Reply:
Hey all I'm going to go ahead and accept this one for now so we can get the basic update live with the maestro redeployment that is being scheduled for tonight, and we can fix the forwarding number with the next update.